Bottom: c66cb3da8347c53e608b652509ec14799a4d726c
Top:    5b3c40765fb3fa7d297372999d94466227ffeacc
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-08 11:38:46 -0500

Refresh of tmp-mantis-database

---
diff --git a/roles/docker-mantisbt/tasks/main.yml b/roles/docker-mantisbt/tasks/main.yml
index 8a83cd8b..08c65221 100644
--- a/roles/docker-mantisbt/tasks/main.yml
+++ b/roles/docker-mantisbt/tasks/main.yml
@@ -84,7 +84,8 @@
     name: '{{ mantisbt_image_name }}'
     tag: '{{ docker_image_tag }}'
     path: '{{ docker_projects_dir }}/docker-mantisbt'
-  
+    force: "{{ docker_image_force_build }}"
+
 # *****************************************************************************
 # Create the data volumes
